src/evidently/options/data_drift.py

```diff
@@ -16,7 +16,14 @@ class DataDriftOptions:
     drift_share: float = 0.5
     nbinsx: Union[int, Dict[str, int]] = DEFAULT_NBINSX
     xbins: Optional[Dict[str, int]] = None
+
     feature_stattest_func: Optional[Union[PossibleStatTestType, Dict[str, PossibleStatTestType]]] = None
+
+    stattest_func: Optional[PossibleStatTestType] = None
```

Comment:
can you add some descriptions in docstrings or comments about the parameters?

Reply:
done